moabb/paradigms/motor_imagery.py

```diff
@@ -58,6 +60,10 @@ def verify(self, dataset):
 
         # we should verify list of channels, somehow
 
+    @abc.abstractmethod
+    def used_events(self, dataset):
```

Comment:
The name is not super clear, i would suggest to rename this function to get_event_id()
but it would be helpful to document this abstract method so we know what we should implement in child class.

Reply:
yeah if you have a better name go ahead and change it, I didn't put too much thought into that. paradigm_specific_events() maybe?

Reply:
i would call it `get_events()` or something like that.

Reply:
sounds good :+1: